tests/unit/tasks/server/middleware/tests-server-test.js

```diff
@@ -20,6 +20,16 @@ describe('TestServerAddon', function () {
       }
     };
 
+    afterEach(function() {
+      nextWasCalled = false;
+      mockRequest = {
+        method: 'GET',
+        path: '',
+        url: 'http://example.com',
+        headers: {}
+      };
+    });
```

Comment:
Assuming I needed to reset state between tests since `mockRequest`'ll get modified inside the middleware (it sets `.path` property specifically, but potentially anything could get dirty). If I was more ambitious, I'd just set everything up in a `beforeEach` block and then wouldn't need the `afterEach`. I think what I have here is the Minimum Viable Change. :)